packages/web/src/components/transaction-details-modal/components/TransactionPurchaseMetadata.tsx

```diff
@@ -0,0 +1,74 @@
+import { useCallback } from 'react'
+
+import { InAppAudioPurchaseMetadata, formatNumberString } from '@audius/common'
+import { IconButton } from '@audius/stems'
+
+import { ReactComponent as IconExternalLink } from 'assets/img/iconExternalLink.svg'
+import {
+  IconUSD,
+  IconSOL,
+  IconAUDIO
+} from 'components/buy-audio-modal/components/Icons'
+
+import { BlockContainer, Block } from './Block'
+import styles from './TransactionPurchaseMetadata.module.css'
+
+const messages = {
+  cost: 'Cost',
+  purchased: 'Purchased',
+  convertedTo: 'Converted To',
+  viewOnSolScan: 'View on Solscan'
+}
+
+export const TransactionPurchaseMetadata = ({
+  metadata
+}: {
+  metadata: InAppAudioPurchaseMetadata
+}) => {
+  const handleSwapSolScanClicked = useCallback(() => {}, [])
+
+  const handlePurchaseSolScanClicked = useCallback(() => {}, [])
+
+  return (
+    <BlockContainer>
+      <Block header={messages.cost}>
+        <IconUSD />
+        {formatNumberString(metadata.usd, { minDecimals: 2, maxDecimals: 2 })}
+      </Block>
+      <Block
+        header={
+          <>
+            {messages.purchased}
+            <IconButton
+              className={styles.iconButton}
+              icon={<IconExternalLink />}
+              title={messages.viewOnSolScan}
+              aria-label={messages.viewOnSolScan}
+              onClick={handlePurchaseSolScanClicked}
```

Comment:
ideally IconButton renders an `<a>`. The way we can make this work, is have our `Button` component accept an href prop, and if it does, render an `<a>` tag (we do this in native button for reference). and then IconButton use Button, (if it's weirdly overloaded, we can have a ButtonBase clase that handles the `a` tag stuff, and Button and IconButton compose it). That is the ideal setup, faster option is to just make iconbutton handle a tag given href prop. lmk if you need help here!